tesseract_core/sdk/cli.py

```diff
@@ -653,6 +654,14 @@ def _display_project_meta(project_id: str) -> list:
         container_ports.append(
             {"name": container.name, "port": host_port, "ip": host_ip}
         )
+        if container.debugpy_port:
+            logger.info(
+                f"Debugpy server listening at http://{host_ip}:{container.debugpy_port}"
```

Comment:
Probably worth renaming the debugpy_port here to `host_debugpy_port` for clarity?